blueprints/ember-cli-eslint/index.js

```diff
@@ -1,9 +1,187 @@
+    var removeJSHintConfig = this._removeJSHintConfig.bind(this);
+
+    if (!this.removePackageFromProject) {
+      return;
+    }
+
+    return this.removePackageFromProject('ember-cli-jshint').then(function() {
+      return removeJSHintConfig();
+    });
+  },
+
+  /**
+   * Find JSHint configuration files and offer to remove them
+   *
+   * @return {RSVP.Promise}
+   */
+  _removeJSHintConfig: function() {
+    var promptRemove = this._promptRemove.bind(this);
+    var removeFile = this._removeFile.bind(this);
+    var ui = this.ui;
+
+    return this._findJSHintConfigFiles()
+      .then(function(files) {
+
+        if (files.length === 0) {
+          ui.writeLine('No JSHint config files found.');
+          return RSVP.resolve({
+            result: {
+              deleteFiles: 'none'
+            }
+          });
+        }
+
+        ui.writeLine('\nI found the following JSHint config files:');
+        files.forEach(function(file) {
+          ui.writeLine('  ' + file);
+        });
+
+        var promptPromise = ui.prompt({
+          type: 'list',
+          name: 'deleteFiles',
+          message: 'What would you like to do?',
+          choices: [
+            { name: 'Delete them all', value: 'all' },
+            { name: 'Delete individually', value: 'each' },
+            { name: 'Delete none', value: 'none' }
+          ]
+        });
+
+        return RSVP.hash({
+          result: promptPromise,
+          files: files
+        });
+      }).then(function(data) {
+        var value = data.result.deleteFiles;
+        var files = data.files;
+
+        // Noop if we're not deleting any files
+        if (value === 'none') {
+          return RSVP.resolve();
+        }
+
+        if (value === 'all') {
+          return RSVP.all(files.map(function(fileName) {
+            return removeFile(fileName);
+          }));
+        }
+
+        if (value === 'each') {
+          return synchronize(files, function(fileName) {
+            return promptRemove(fileName);
+          });
+        }
+      });
+  },
+
+  /**
+   * Find JSHint configuration files
+   *
+   * @return {Promise->string[]} found file names
+   */
+  _findJSHintConfigFiles: function() {
+    var projectRoot = this.project.root;
+    var ui = this.ui;
+
+    ui.startProgress('Searching for JSHint config files');
+    return new RSVP.Promise(function(resolve) {
+      var files = walkSync(projectRoot, {
+        globs: ['**/.jshintrc'],
+        ignore: [
+          'bower_components',
+          'dist',
+          'node_modules',
+          'tmp'
+        ]
+      });
+
+      ui.stopProgress();
+      resolve(files);
+    });
+  },
+
+  /**
+   * Prompt the user about whether or not they want to remove the given file
+   *
+   * @param {string} filePath the path to the file
+   * @return {RSVP.Promise}
+   */
+  _promptRemove: function(filePath) {
+    var removeFile = this._removeFile.bind(this);
+    var message = 'Should I remove `' + filePath + '`?';
```

Comment:
up above you talk about deleting, here it's removing. I think we should probably try to stay consistent here 😉